src/Transpiler.ts

```diff
@@ -917,8 +917,12 @@ export class LuaTranspiler {
                 return this.transpileArrayCallExpression(node);
             }
 
-            if (expType.symbol && (expType.symbol.flags & ts.SymbolFlags.Namespace)) {
+            const expType2 = this.checker.getTypeAtLocation(node.expression);
```

Comment:
It might be worth renaming `expType` and `expType2` to something more descriptive - such as `containerType` and `functionType` respectively.

Reply:
agreed